tools/install.py

```diff
@@ -226,6 +226,12 @@ def wanted_v8_headers(files_arg, dest):
     files_arg = [name for name in files_arg if name in v8_headers]
     action(files_arg, dest)
 
+  def wanted_zoslib_headers(files_arg, dest):
+    import glob
+    zoslib_headers = glob.glob(zoslibinc + '/*.h')
```

Comment:
Consideration for the future: We may want to change this to be recursive: https://docs.python.org/3/library/pathlib.html#pathlib.Path.rglob

Such that subdirectories are included, which will be introduced in: https://github.ibm.com/open-z/zoslib/pull/43/files if approved.